atc/exec/artifact_input_step.go

```diff
@@ -56,7 +59,8 @@ func (step *ArtifactInputStep) Run(ctx context.Context, state RunState) error {
 		return ArtifactVolumeNotFoundError{buildArtifact.Name()}
 	}
 
-	workerVolume, found, err := step.workerClient.FindVolume(logger, volume.TeamID(), volume.Handle())
+	// TODO-Later artifact_input_step shouldn't be looking up the volume on the worker
```

Comment:
do you mind expanding on the eventual plan for this?

Reply:
#3607